File: Java/FactorialNumber.java
import java.util.Scanner;

public class FactorialNumber 
{
    public static void main(String[] args) 
    {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number : ");
        int n = scanner.nextInt();

        int fact = 1;

        for (int i = 1; i <= n; i++) 
        {
            fact = fact * i;
        }

        System.out.println("The factorial of " + n + " is " + fact);
        scanner.close();
    }
}

File: C#/FactorialNumber.cs
//Write a c# program to print factorial of a number.

using System;
public class factorialNumber
{
    static void Main()
    {
        int n;
        Console.WriteLine("Enter the number : ");
        n = int.Parse(Console.ReadLine());

        int fact = 1;

        for (int i = 1; i <= n; i++)
        {
            fact = fact * i;
        }

        Console.WriteLine("The factorial of {0} is " + fact, n);
        Console.ReadLine();
    }
}